pkgs/applications/networking/remote/freerdp/unstable.nix

```diff
@@ -6,6 +6,7 @@
 }:
 
 stdenv.mkDerivation rec {
+  version = ";";
```

Comment:
(accidentally did `git commit -am` on an unstaged local change ... reverting this line)